### PR TITLE
First set of articles

### PR DESCRIPTION
**Pull request by igortg**
List is sorted in order of complexity

### CODE REVIEW

**Review by kfasolin — APPROVED** [review]
Awesome initiative! 😄

**Comment by marcelotrevisani** [comment]
It would be good if we create like a new section for talks and books as well
:)

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
 # Awesome Dev Articles [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
-A curated list of articles
+A curated list of articles created by the ESSS Dev team
```
Comment (nicoddemus) [inline README.md]: "created"? We didn't write those articles

File: README.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+
+* [Steps to better code](https://medium.com/@isaaclyman/steps-to-better-code-e6c3cce0c7f9) - Isaac Lyman
+* [The Ugly American Programmer](https://blog.codinghorror.com/the-ugly-american-programmer/) - Jeff Atwood
+* [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) - Chris Beams
+* [The Opposite of Fitts' Law](https://blog.codinghorror.com/the-opposite-of-fitts-law/) - Jeff Atwood
+* [Empty - The Codeless Code](http://thecodelesscode.com/case/6) - Qi
+* [Things You Should Never Do, Part I](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/) - Joel Spolsky
+* [The Joel Test: 12 Steps to Better Code](https://www.joelonsoftware.com/2000/08/09/the-joel-test-12-steps-to-better-code/) - Joel Spolsky
+* [Move Fast and Fix Things](https://githubengineering.com/move-fast/) - Vicent Martí
+* [Notes on structured concurrency, or: Go statement considered harmful](https://vorpus.org/blog/notes-on-structured-concurrency-or-go-statement-considered-harmful/) -  Nathaniel J. Smith
```
Comment (nicoddemus) [inline README.md:21]: Please add a section "how to add new articles":

```markdown
### How to add new articles

The idea of this list is to aggregate articles and interesting links that we usually share in the mailing list/RocketChat for easier reference in the future, so new articles are welcome!

To add a new article just open a new PR against the repository, adding the link in the correct position to keep the list sorted (you can open the PR directly in GitHub's UI).
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
 # Awesome Dev Articles [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
-A curated list of articles
+A curated list of articles created by the ESSS Dev team
 
 Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 ## Articles
+
+* [Steps to better code](https://medium.com/@isaaclyman/steps-to-better-code-e6c3cce0c7f9) - Isaac Lyman
```
Comment (nicoddemus) [inline README.md]: How about sorting this by title or date?

Comment (igortg) [reply README.md]: It's sorted by complexity. My intention was to make it easier for newcomers to pick what to read. Maybe we could sort by title and add sections "Basic" and "Advanced"

Comment (nicoddemus) [reply README.md]: Sounds good.

**Review by tadeu — APPROVED** [review]

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**